models/datasets/README.md

```diff
@@ -0,0 +1,150 @@
+
+### Example Data for Developer Guide
+Additionally a subset of 100 messages from the dataset were augmented to include sender and recipient information using the python [faker](https://faker.readthedocs.io/en/master/#) package. If there is any resemblance to real individuals, it is purely coincidental.
+- [emails_with_addresses.jsonlines](../../examples/data/email_with_addresses.jsonlines)
+
+
+## Ransomware Detection via AppShield
+
+The dataset was generated by running ransomware and benign processes in a lab environment and recording the output from several plugins from the [Volatility framework](https://github.com/volatilityfoundation/volatility3) including `cmdline`, `envars`, `handles`, `ldrmodules`, `netscan`, `pslist`, `thredlist`, `vadinfo`. The training csv file contains 530 columns- a combination of features from the Volatility Plugins. This data collection is part of [DOCA AppShield](https://developer.nvidia.com/networking/doca).
+
+### Sample Training Data
+Training data csv consists of 87968 preprocessed and labeled AppShield processes from 32 snapshots collected from 256 unique benign and ransomware activities.
+- [ransomware-training-data.csv](./training-data/ransomware-training-data.csv)
+
+### Pipeline Validation Data
+The validation set contains raw data from 27 AppShield snapshots.
+- [appshield data directory](../../examples/data/appshield/Heur)
+
+## Root Cause
+
+This dataset contains a small sample of sanitized linux kernel logs of a DGX machine prior to a hardware failure. The training dataset contains 1359 logs labeled as indicators of the root cause or not. A model trained on this set can be robust enough to correctly identify previously unseen errors from the `unseen-errors` file as a root cause as well. 
+
+### Sample Training Data
+- [root-cause-training-data.csv](./training-data/root-cause-training-data.csv)
+- [root-cause-unseen-errors.csv](./training-data/root-cause-unseen-errors.csv)
+
+### Pipeline Validation Data
+- [root-cause-validation-data-input.jsonlines](./validation-data/root-cause-validation-data-input.jsonlines)
+
+
+## Sensitive Information Detection (SID)
+
+This data contains 2000 synthetic pcap payloads generated to mimic sensitive and benign data found in nested jsons from web APIs and environmental variables. Each row is labeled for the presence or absence of 10 different kinds of sensitive information. The data was generated using the python [faker](https://faker.readthedocs.io/en/master/#) package and lists of most [common passwords](https://github.com/danielmiessler/SecLists/tree/master/Passwords/Common-Credentials). If there is any resemblance to real individuals, it is purely coincidental.
+
+### Sample Training Data
+- [sid-sample-training-data.csv](./training-data/sid-sample-training-data.csv)
+
+### Pipeline Validation Data
+- [sid-validation-data.csv](./validation-data/sid-validation-data.csv)
```

Comment:
Let's add a section at the bottom that makes it clear that these are for testing/evaluation purposes only and not meant for production. Maybe something like:

Morpheus contributors will make every effort to keep datasets up-to-date and accurate. However, the data submitted to this repository is provided on an “as is” basis and there is no warranty or guarantee of any kind that the information is accurate, complete, current or suitable for any particular purpose. It is the responsibility of all persons who use Morpheus to independently confirm the accuracy of the data, information, and results obtained via the Morpheus example workflows.

Reply:
added Disclaimer section at the bottom